src/Tests/AuthoringTest/AuthoringTest.csproj

```diff
@@ -7,6 +7,7 @@
     <!-- Enable to diagnose generation issues -->
     <!-- <CsWinRTEnableLogging>true</CsWinRTEnableLogging> -->
     <!-- <CsWinRTKeepGeneratedSources>true</CsWinRTKeepGeneratedSources> -->
+    <CsWinRTIIDOptimizerOptOut>true</CsWinRTIIDOptimizerOptOut>

```

Comment:
why are we opting out?

Reply:
There is an invalid cast when using the patcher on the AuthoringTest, I am investigating. I think it may be related to the [invalid cast on ICustomProperty](https://github.com/microsoft/CsWinRT/issues/856)